===================================================================
RCS file: /cvs/gnome/gtk+/gdk/gdkpango.c,v
retrieving revision 1.44.2.1
retrieving revision 1.44.2.2
diff -u -r1.44.2.1 -r1.44.2.2
--- gdkpango.c	2006/02/20 18:00:36	1.44.2.1
+++ gtk+/gdk/gdkpango.c	2006/04/28 07:05:12	1.44.2.2
@@ -236,12 +236,12 @@
     {
       cairo_save (cr);
       emboss_context (cr);
-      cairo_move_to (cr, x / PANGO_SCALE, y / PANGO_SCALE);
+      cairo_move_to (cr, (double)x / PANGO_SCALE, (double)y / PANGO_SCALE);
       pango_cairo_show_glyph_string (cr, font, glyphs);
       cairo_restore (cr);
     }
   
-  cairo_move_to (cr, x / PANGO_SCALE, y / PANGO_SCALE);
+  cairo_move_to (cr, (double)x / PANGO_SCALE, (double)y / PANGO_SCALE);
   pango_cairo_show_glyph_string (cr, font, glyphs);
 }
 
